.pre-commit-config.yaml

```diff
@@ -43,7 +43,7 @@ repos:
     rev: v0.32.0
     hooks:
       - id: yapf
-        args: ['--in-place', '--parallel']
+        args: ['--in-place', '--parallel', '--recursive']
```

Comment:
If I remember correctly, `--recursive` isn't needed here.

`pre-commit` applies changes to all files included in the commit (`git diff` / `git show --stat`).

Reply:
Just do `pre-commit run --all-files` once locally to apply it to all files in the repository, that should be enough.

Reply:
done